docs/book/src/developer/tilt.md

```diff
@@ -35,19 +35,23 @@ You can see the status of the cluster with:
 kubectl cluster-info --context kind-capi-test
 ```
 
-### Create a tilt-settings.json file
-
-Next, create a `tilt-settings.json` file and place it in your local copy of `cluster-api`. Here is an example:
-
-```json
-{
-  "default_registry": "gcr.io/your-project-name-here",
-  "provider_repos": ["../cluster-api-provider-aws"],
-  "enable_providers": ["aws", "docker", "kubeadm-bootstrap", "kubeadm-control-plane"]
+### Create a tilt-settings.yaml file
```

Comment:
```suggestion
### Create a tilt-settings.yaml or tilt-settings.json file
```
I think we should still surface that both is possible. There are also reasons why folks might prefer JSON files (not prone to indentation errors, keys can also be "commented-out" by prefixing them with "_", ...)

It could also make folks wonder why their json files work.

(I would also add a note as `<aside>` at the end of this section that json is possible too)

Reply:
my two cents
```suggestion
### Create a tilt-settings file
```
and then add a sentence that it also possible to provider setting in json format, and that yaml take precedence if both exist

Reply:
Fine for me